opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/export/in_memory_log_exporter.py

```diff
@@ -16,11 +16,14 @@
 import typing
 
 from opentelemetry.sdk._logs._internal import ReadableLogRecord
-from opentelemetry.sdk._logs.export import LogExporter, LogExportResult
+from opentelemetry.sdk._logs.export import (
+    LogRecordExporter,
+    LogRecordExportResult,
+)
 
 
-class InMemoryLogExporter(LogExporter):
```

Comment:
I think we are missing the deprecated version of this to not break our own genai instrumentations